### DISCUSSION TITLE
IAM Access Analyzer: false positive

### DISCUSSION THREAD

**Discussion by Timost** [Checkmarx/kics#6533] [Q&A] [answered]
Hi,
Thank you for Kics ! 🙏

With yesterday's v1.7.4 release, I have what I think are false positives on the [IAM Access Analyzer query](https://docs.kics.io/latest/queries/cloudformation-queries/aws/8d29754a-2a18-460d-a1ba-9509f8d359da/).

I have an ansible code base where all files raise this warning.

```
7:28AM WRN Failed to detect line, query response Resources fileName=redacted.yml queryName=iam_access_analyzer_not_enabled scanID=console

...

Files scanned: 319
Parsed files: 294
Queries loaded: 1893
Queries failed to execute: 0
------------------------------------
IAM Access Analyzer Not Enabled, Severity: LOW, Results: 173
Description: IAM Access Analyzer should be enabled and configured to continuously monitor resource permissions
Platform: CloudFormation
Learn more about this vulnerability: https://docs.kics.io/latest/queries/cloudformation-queries/aws/8d29754a-2a18-460d-a1ba-9509f8d359da
```

**Comment by cx-marco-pereira** [comment, marked as answer]
There were indeed a lot of false positives related to this query, as answered in [this](https://github.com/Checkmarx/kics/issues/6549) issue, Every false positive was causing the warning couldn't detect line to be thrown and it should be fixed with the linked PR 😄 

Thank you for the feedback and for using KICS

**Reply by Timost** [reply]
Thank you for your answer ! 🙏

I opened a discussion and not an issue sorry if that was not the proper place. '^^
